content/_data/upgrades/2-346-1.adoc

```diff
@@ -0,0 +1,30 @@
+==== PNG and GIF icons replaced with SVG icons
+
+Scalable vector graphics images are used in Jenkins instead of the GIF and PNG images that were used previously.
+SVG images provide a better user experience and improve the Jenkins user experience.
+
+The GIF and PNG images have been removed from Jenkins core.
+
+Jenkins has removed a large number of GIF and PNG iamges.
+Ensure all your plugins are the most recent release before updating core, otherwise images may be missing.
+Check for new plugin versions immediately after the core update and install available updates.
+
+==== Java Native Runtime library removed
+
+The Java Native Runtime (JNR) library has been removed from Jenkins core.
+Users of https://plugins.jenkins.io/pam-auth/[PAM Authentication], https://plugins.jenkins.io/file-leak-detector/[CloudBees File Leak Detector], and https://plugins.jenkins.io/maven-repo-cleaner/[Maven Repository Scheduled Cleanup] must upgrade those plugins to compatible versions.
+
+==== Self-signed certificate not provided automatically
+
+In previous versions of Jenkins, a self-signed certificate was used if no HTTPS keystore was specified when processing HTTPS inside the Jenkins java process.
+This feature has been removed.
+
+If HTTPS or HTTP/2 is being processed directly by the Jenkins java process, an HTTPS keystore must be configured by adding the `--httpsKeyStore=path/to/keystore` and `--httpsKeyStorePassword=keystorePassword` arguments.
+This change does not affect HTTPS processing that is performed in link:/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/[reverse proxies] like Nginx and Apache HTTP Server.
+
+See link:/doc/book/installing/initial-settings/#https-with-an-existing-certificate[HTTPS with an existing certificate] and link:/doc/book/installing/initial-settings/#https-certificates-with-windows[HTTPS certificates with Windows] for more information on creating a key store.
+
+==== Reject connections from unsupported agent versions
+
+Jenkins now rejects connections from agents with unsupported Remoting versions.
+This can be disabled by setting the `hudson.slaves.SlaveComputer.allowUnsupportedRemotingVersions` system property to true.
```

Comment:
What does this mean? How can admins tell that this is happening? How can they fix it?

Reply:
I've done some exploratory testing and added a description of the results of that testing as 674a76b2d55d419bafa6354ec57228981a9f311f

I've also submitted [JENKINS-68785](https://issues.jenkins.io/browse/JENKINS-68785) to report that I didn't see a log message when an unsupported remoting version was disconnected from the controller.